source/codegen/metadata/niscope/attributes.py

```diff
@@ -1,2556 +1,1558 @@
+        'grpc_type': 'sint32',
         'name': 'ACQUISITION_TYPE',
         'resettable': True,
         'type': 'ViInt32'
     },
     1250102: {
-        'access': 'read-write',
-        'channel_based': False,
-        'documentation': {
-            'description': '\nConfigures the device to automatically complete an acquisition if a trigger has not been received.\nValid Values:\nNone (1)         - Normal triggering\nAuto Trigger (2) - Auto trigger acquisition if no trigger arrives\n'
-        },
+        'codegen_method': 'public',
         'enum': 'TriggerModifier',
-        'lv_property': 'Triggering:Trigger Modifier',
+        'grpc_type': 'sint32',
         'name': 'TRIGGER_MODIFIER',
         'resettable': True,
         'type': 'ViInt32'
     },
     1250103: {
-        'access': 'read-write',
-        'channel_based': True,
-        'documentation': {
-            'description': '\nSpecifies the input impedance for the channel in Ohms.\n'
-        },
+        'codegen_method': 'public',
         'enum': 'InputImpedance',
-        'lv_property': 'Vertical:Input Impedance',
+        'grpc_type': 'double',
         'name': 'INPUT_IMPEDANCE',
         'resettable': True,
         'type': 'ViReal64'
     },
     1250106: {
-        'access': 'read only',
-        'channel_based': False,
-        'documentation': {
-            'description': '\nIndicates the sample mode the digitizer is currently using.\n'
-        },
+        'codegen_method': 'public',
         'enum': 'SampleMode',
-        'lv_property': 'Acquisition:Sample Mode',
+        'grpc_type': 'sint32',
         'name': 'SAMPLE_MODE',
         'resettable': False,
         'type': 'ViInt32'
     },
     1250109: {
-        'access': 'read-write',
-        'attribute_class': 'AttributeViReal64TimeDeltaSeconds',
-        'channel_based': False,
-        'documentation': {
-            'description': '\nSpecifies the length of time from the trigger event to the first point in the waveform record in seconds.  If the value is positive, the first point in the waveform record occurs after the trigger event (same as specifying NISCOPE_ATTR_TRIGGER_DELAY_TIME).  If the value is negative, the first point in the waveform record occurs before the trigger event (same as specifying NISCOPE_ATTR_HORZ_RECORD_REF_POSITION).\n'
-        },
-        'lv_property': 'Horizontal:Advanced:Acquisition Start Time',
+        'codegen_method': 'public',
+        'grpc_type': 'double',
         'name': 'ACQUISITION_START_TIME',
         'resettable': True,
-        'type': 'ViReal64',
-        'type_in_documentation': 'hightime.timedelta, datetime.timedelta, or float in seconds'
+        'type': 'ViReal64'
     },
     1250201: {
-        'access': 'read-write',
-        'channel_based': False,
-        'documentation': {
-            'description': '\nSpecifies the type of video signal, such as NTSC, PAL, or SECAM.\n'
-        },
-        'enum': 'VideoSignalFormat',
```

Comment:
Missing 'enum' tag for this attribute. You can look [here](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/358221?_a=files&path=/src/nidmm/nidmmBaseDistribution/nidmmAPIMetadata/source/nidmmAPIMetadata/grpc_device/attributes/attributes.hapigen) for an example of how to add these tags.